soc/riscv/openisa_rv32m1/soc_context.h

```diff
@@ -24,12 +24,12 @@
 
 /* Initial saved state. */
 #define SOC_ESF_INIT						\
```

Comment:
I am unsure why this initially assigned `0xdeadband` here, it seems to have been the default context init value. But it wasn't really used to initialize the extended stack frame before. I tried to align it with the intention according to the documentation in https://github.com/zephyrproject-rtos/zephyr/blob/master/arch/riscv/Kconfig#L51 .